generator/templates/Default/src/plugins/vuetify/theme.js

```diff
@@ -0,0 +1,9 @@
+const light = {
+  primary: '#2A73C5',
+};
+
+const dark = {
+  primary: '#2A73C5',
+};
```

Comment:
Lucky i saw this.
Those are not the same color as they where.
The previous colors are the Kings Code colors and the default for the scaffold.